manifest.json

```diff
@@ -25,6 +25,8 @@
   },
   "permissions": [
     "tabs",
-    "https://filtr.news/*"
+    "https://filtr.news/*",
```

Comment:
Ahja. I just removed them haha. so they were actually needed. poor testing, my bad. But i think the filtr.news can be removed as we have the wildcard matches in place